docs/getting-started.md

```diff
@@ -22,65 +22,103 @@ Install and run the [Semgrep command-line interface](https://github.com/returnto
-        {label: 'Docker', value: 'Docker'},
-        ]}
-    >
-
-    <TabItem value='macOS'>
+To install and run Semgrep OSS Engine, use one of the following options:
 
-    ```bash
-    brew install semgrep
-    ```
+<Tabs
+    defaultValue="macOS"
+    values={[
+    {label: 'macOS', value: 'macOS'},
+    {label: 'Linux', value: 'Linux'},
+    {label: 'Windows Subsystem for Linux (WSL)', value: 'Windows Subsystem for Linux (WSL)'},
+    {label: 'Docker', value: 'Docker'},
+    ]}
+>
 
-    Alternatively:
+<TabItem value='macOS'>
 
-    ```bash
-    python3 -m pip install semgrep
-    ```
+  1. Install:
+      ```bash
+      brew install semgrep
+      ```
 
-    </TabItem>
+      Alternatively:
 
-    <TabItem value='Linux'>
+      ```bash
+      python3 -m pip install semgrep
+      ```
 
-    ```bash
-    python3 -m pip install semgrep
-    ```
+  2. Confirm installation by the following command:
+      ```sh
+      semgrep --version
+      ```
+  3. Run recommended Semgrep Registry rules:
+      <pre class="language-bash"><code>semgrep --config=auto <span className="placeholder">PATH/TO/SRC</span></code></pre>
+      Substitute the optional placeholder <code><span className="placeholder">PATH/TO/SRC</span></code> with the path to your source code.
 
-    </TabItem>
 
-    <TabItem value='Windows Subsystem for Linux (WSL)'>
+</TabItem>
 
-    ```bash
-    python3 -m pip install semgrep
-    ```
+<TabItem value='Linux'>
+
+  1. Install:
+      ```bash
+      python3 -m pip install semgrep
+      ```
+
+  2. Confirm installation by the following command:
+      ```sh
+      semgrep --version
+      ```
+
+  3. Run recommended Semgrep Registry rules:
+      <pre class="language-bash"><code>semgrep --config=auto <span className="placeholder">PATH/TO/SRC</span></code></pre>
+      Substitute the optional placeholder <code><span className="placeholder">PATH/TO/SRC</span></code> with the path to your source code.
 
-    </TabItem>
 
-    <TabItem value='Docker'>
+</TabItem>
 
+<TabItem value='Windows Subsystem for Linux (WSL)'>
+
+  1. Install:
+      ```bash
+      python3 -m pip install semgrep
+      ```
+
+  2. Confirm installation by the following command:
+      ```sh
+      semgrep --version
+      ```
+
+  3. Run recommended Semgrep Registry rules:
+      <pre class="language-bash"><code>semgrep --config=auto <span className="placeholder">PATH/TO/SRC</span></code></pre>
+      Substitute the optional placeholder <code><span className="placeholder">PATH/TO/SRC</span></code> with the path to your source code.
+
+
+</TabItem>
+
+<TabItem value='Docker'>
+
+  1. Pull latest image locally:
+     ```sh
+     docker pull returntocorp/semgrep
+     ```
+   
+  2. Confirm version by the following command:
+      ```sh
+      docker run --rm returntocorp/semgrep semgrep --version
+      ```
+
+  3. Run recommended Semgrep Registry rules:
     ```bash
     docker run --rm -v "${PWD}:/src" returntocorp/semgrep semgrep --config=auto
     ```
```

Comment:
```suggestion
    <pre class="language-bash"><code>docker run --rm -v "${PWD}:<span className="placeholder">/SRC</span>" returntocorp/semgrep semgrep --config=auto</code></pre>

```

Reply:
@armchairlinguist This will not show a proper code highlighting, but it will display the SRC as a placeholder (italics, changed color also). This approach is inspired by https://developers.google.com/style/placeholders - but they also have some JavaScript for users to write the code immediately in the placeholder, and then copy it with their changed placeholder.